### PR TITLE
No filter validation

### PR DESCRIPTION
**Pull request by bog**
This PR removes filter validation and option contexts (which would have paved the way for more validation there). In the name of simplicity, we're no longer running ffmpeg when the first FilterNode is instantiated to get filter information.

Why?
ffmpeg is good at this validation. let it be so we don't need to be.

How do we connect chains now?
filter nodes gain output pads when a streamSpecifier is requested 
(otherwise, we assume outputs go to the next chain or output).

### CODE REVIEW

**Comment by nihonjinrxs** [comment]
Do we expect this change to break node 8 support? [That build is failing.](https://travis-ci.com/tedconf/fessonia/jobs/274403435)

File: lib/ffmpeg_option.js [hunk 1]
```diff
@@ -90,49 +79,32 @@ class FFmpegOption {
    * @returns {string} the string representation
    */
   [util.inspect.custom](depth, opts) {
-    const context = `FFmpegOption.FFmpegOptionContexts.${['GLOBAL', 'INPUT', 'OUTPUT'][this.context]} (=== ${this.context})`;
-    return util.inspect({
-      ...this,
-      context: context
-    }, false, depth, opts.color ? opts.color : true);
+    return util.inspect(this, false, depth, opts.color ? opts.color : true);
   }
```
Comment (nihonjinrxs) [inline lib/ffmpeg_option.js]: Looks like something about this change is causing a recursion loop in node 8.

**Comment by nihonjinrxs** [comment]
closes #24

**Comment by bog** [comment]
> That's one way to fix it. It still concerns me a bit that the custom inspects were there before and worked in node v8. Do we know what in here made them stop working properly? I'm worried this is just hiding something we should really be looking at.

Pretty sure the custom inspect stuff was causing it to try to print circular references (which were new). Default logging avoids that. Not sure what changed in v10 to make that not happen.

**Pull request merged**